### ISSUE TITLE
src toolchain add should check current dir before adding

### ISSUE THREAD

**Issue by sqs** [sourcegraph/srclib#98]
`src toolchain add <TC>` should check that the current dir is indeed a toolchain before adding. If it is not a toolchain dir, then a lot of other operations fail with bizarre errors about symlinks.

**Comment by jelmerdereus** [comment]
@samer: "ConfigFilename is the file that all toolchains need"

**Comment by samertm** [comment]
Yup, and the check should be here: https://sourcegraph.com/sourcegraph.com/sourcegraph/srclib@1d19551120a015de42e0cf648bdfa9ddf4061986/.tree/toolchain/add.go#startline=35&endline=35

**Comment by samer** [comment]
@jelmerdereus wrong Samer :-)

**Comment by jelmerdereus** [comment]
excuse me @samer😄

**Comment by jelmerdereus** [comment]
Please have a look @samertm
